cms/templates/widgets/tabs/metadata-edit-tab.html

```diff
@@ -13,5 +14,10 @@
         <%static:include path="js/${template_name}.underscore" />
     </script>
 % endfor
-
-<div class="wrapper-comp-settings metadata_edit" id="settings-tab" data-metadata='${json.dumps(editable_metadata_fields) | h}'></div>
+<% metadata_field_copy = copy.copy(editable_metadata_fields) %>
```

Comment:
Could the disabled field be checked instead of specifying thumbnail specifically?

Reply:
What do you mean?

Reply:
Since you're setting up the thumbnail field with `disabled=True`, could you check editable_metadata_fields for this?

Reply:
oh thank you for catching that! `disabled=true` doesn't do anything, it is left over from before I found this solution. There is no way to disable a field, so in other code that I found in edx-platform, they just made a copy of the fields and deleted it from the copy.

Reply:
Ah I see. Everything else looks good to me!